diktat-cli/src/test/kotlin/org/cqfn/diktat/util/CliUtilsKtTest.kt

```diff
@@ -0,0 +1,92 @@
+package org.cqfn.diktat.util
+
+import org.jetbrains.kotlin.konan.file.File
```

Comment:
Are you sure this is not `java.io.File`?

Reply:
yeap, chnaged